commons-math-legacy-core/src/main/java/org/apache/commons/math4/legacy/core/jdkmath/AccurateMath.java

```diff
@@ -1600,9 +1600,8 @@ public static double pow(final double x, final double y) {
                     z = z * lnb + 1.0;
                     z *= lnb;
 
-                    final double result = exp(lna, z, null);
                     //result = result + result * z;
-                    return result;
+                    return exp(lna, z, null);
```

Comment:
I am wondering about the cryptic commented out code here. It would be run after the result has been computed with `exp`. So if you simply return the result of `exp` then the comment is before the `exp` and out of context. I would leave this unchanged until the reason for commenting out the final computation is clarified.

Reply:
HI @aherbert 
Sound logical. I made rollback of this change.
TY